.pre-commit-config.yaml

```diff
@@ -0,0 +1,30 @@
+fail_fast: true
+
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: debug-statements
+
+-   repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+    -   id: black
+        args: [--line-length=79]
+
+-   repo: https://github.com/pycqa/flake8
+    rev: 4.0.1
+    hooks:
+    -   id: flake8
```

Comment:
Can we also move the args for flake8 in `setup.cfg` here?

Reply:
(then I'll stop nagging, I promise 😆)

Reply:
No problem. There's a lot of things I don't know about, so keep on nagging if you want :-)